src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Migrations/OpenIdMigrations.cs

```diff
@@ -53,23 +66,26 @@ public int Create()
                 .Column<string>(nameof(OpenIdAuthorizationIndex.Status))
                 .Column<string>(nameof(OpenIdAuthorizationIndex.Subject))
                 .Column<string>(nameof(OpenIdAuthorizationIndex.Type))
-                .Column<DateTime>(nameof(OpenIdAuthorizationIndex.CreationDate)));
+                .Column<DateTime>(nameof(OpenIdAuthorizationIndex.CreationDate)),
+                collection: OpenIdAuthorizationCollection);
 
             SchemaBuilder.AlterIndexTable<OpenIdAuthorizationIndex>(table => table
-                .CreateIndex("IDX_OpenIdAuthorizationIndex_DocumentId_ApplicationId",
+                .CreateIndex("IDX_COL_OpenIdAuth_DocumentId_ApplicationId",
```

Comment:
@jtkech @Skrypt 

So I invent a new convention here (always subject to change)
where I use
`IDX_COL` to indicate Index and Collection, and then an abbreviated name of the index class (to get some reduction in indexes, and avoid mysql / postrgres issues).

Needs to be different to the existing index names, because both must co-exist for a time during the migration.

Reply:
@deanmarcussen 

`IDX_COL_OpenIdAuth_DocumentId_ApplicationId` LGTM for a new and shorter name

Hmm, but now we know that we always include `DocumentId`, so another idea would be

`IDX_COL_OpenIdAuthorization_ApplicationId`

Hmm, maybe with a shorter standardized prefix for indexes + collection

`IDXC_OpenIdAuthorization_ApplicationId`

But anyway seems to be too late ;)